pkg/disk/disk.go

```diff
@@ -121,7 +122,15 @@ func (m *userDataDiskManager) createLimaDisk() error {
 func (m *userDataDiskManager) attachPersistentDiskToLimaDisk() error {
 	limaPath := fmt.Sprintf("%s/_disks/%s/datadisk", m.finch.LimaHomePath(), diskName)
 	if !m.persistentDiskExists() {
-		err := m.fs.Rename(limaPath, m.finch.UserDataDiskPath(m.homeDir))
+		disksDir := path.Dir(m.finch.UserDataDiskPath(m.homeDir))
```

Comment:
This new logic should handle the case where the dir doesn't exist and migrating an old datadisk to the new name right?

Reply:
it does handle the case where the dir doesn't exist, yeah.

> migrating an old datadisk to the new name

this isn't included currently. I'm a little torn on whether to include it. perhaps we can put a notice on the release that you will lose your data upgrading from 0.2.0 to 0.2.1, with manual steps on how to restore it?

Reply:
Yeah, I think that makes sense. We don't really want to carry that migration code. Not sure if that means it needs to be 0.3.0 instead of 0.2.1, maybe @ningziwen can chime in?

Reply:
I agree. It should be sufficient if we can put a notice in the release note. 

As this is a fix change, I guess it should be 0.2.1?